app/views/layouts/angular/_form_buttons_verify_angular.html.haml

```diff
@@ -4,9 +4,9 @@
 - else
   - verify_title_on = _("Validate the credentials by logging into the Server")
   - verify_title_off ||= _("Server information, Username and matching password fields are needed to perform verification of credentials")
-- if controller.send(:restful?)
+- if controller.send(:restful?) || controller.send(:validate_before_save?)
   -# TODO: replace first argument with a sane way of submitting the form
-  - validate = "#{main_scope}.validateClicked({target: '.validate_button:visible'}, '#{valtype}', true)"
+  - validate = "#{main_scope}.validateClicked({target: '.validate_button:visible'}, '#{valtype}', true, angularForm, '#{url_for_only_path(:action => validate_url, :id => id, :type => valtype, :button => "validate")}')"
```

Comment:
This looks wrong. Why does `miqService` need to set the auth status?

Feels like we should have a nice `vm.validationStatus = true` (or `vm.whateverModel.valid` maybe), and just set that and let angular do it's thing. 

I guess another future refactoring :(